.azure-pipelines/templates/jobs/packaging-jobs.yml

```diff
@@ -100,3 +125,32 @@ jobs:
       - script: |
           python -m tox -e integration-external,apacheconftest-external-with-pebble
         displayName: Run tox
+  - job: snap_dns_run
+    dependsOn: snap_dns_build
```

Comment:
Shouldn't this also depend on `snap_build`?

Reply:
Yes you are right.